components/cam/bld/namelist_files/namelist_defaults_cam.xml

```diff
@@ -1668,10 +1681,20 @@
 <se_nsplit dyn_target="preqx" hgrid="ne0np4_twpx4v1" > 4 </se_nsplit>
 <se_nsplit dyn_target="preqx"> 2 </se_nsplit>
 
+<!-- ne45 defaults to new time step interface -->
+<se_tstep hgrid="ne45np4">  200 </se_tstep>
```

Comment:
se_tstep was already set above

Reply:
good, I'll move these lines around t be consistent with how everything is grouped.